sentry-ruby/lib/sentry-ruby.rb

```diff
@@ -249,10 +244,16 @@ def close
         @backpressure_monitor = nil
       end
 
-      if configuration&.include_local_variables
-        exception_locals_tp.disable
+      if client = get_current_client
```

Comment:
@sl0thentr0py Another "concern" I have is that we currently only focus on the main hub's top client, but not other clients. Ideally we should do that for all clients of the main hub, as well as other hubs' (which we don't track AFAIK) clients?

I think the former is probably fine as a hub is unlikely to have more than 1 client. But I do recall some customers tried doing multi-hub setup in the past (also rare tho).

I know both are fairly rare. I just want to raise it so we keep that in mind.

Reply:
yea both valid concerns but as a heads up, sometime next year we'll move to removing the Hub and only having a Scope (which will be closer to opentelemetry's context). 
rfc here - https://github.com/getsentry/rfcs/pull/122

Reply:
Thank you for the context! I just gave it a first pass and got confused around different types of scopes too, as raised by you and many other comments.

But yeah if I understand it correctly, there will only be 1 client in the future and no hubs, which means this won't be a concern anymore.